invenio_rdm_records/theme/templates/invenio_rdm_records/details/contributors.html

```diff
@@ -0,0 +1,21 @@
+{#
+    Copyright (C) 2020 CERN.
+    Copyright (C) 2020 Northwestern University.
+  
+    Invenio RDM Records is free software; you can redistribute it and/or modify
+    it under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- for group in record.contributors|groupby('role')%}
+<h5><b>{{group.grouper|contributortype_title}}(s)</b></h5>
+
+{%- for creator in group.list %}
```

Comment:
`creator` -> `contributor` or we can use macro to reuse across contributor and creators.

Reply:
I didnt create a macro cuz it felt a bit overkill, its true that is repeated code. But also I am not sure how the UI will evolve. I guess we will be creating macros in the next iterations when we have more and more UI :)

changed to contributor :)